tiledb/sm/config/config.cc

```diff
@@ -478,6 +487,12 @@ Status Config::unset(const std::string& param) {
         VFS_S3_AWS_SECRET_ACCESS_KEY;
   } else if (param == "vfs.s3.aws_session_token") {
     param_values_["vfs.s3.aws_session_token"] = VFS_S3_AWS_SESSION_TOKEN;
+  }  else if (param == "vfs.s3.aws_session_token") {
```

Comment:
look at this path again, the if-statement strings don't seem to match the config string names in the assignments